wafer/settings.py

```diff
@@ -148,6 +148,8 @@
     'wafer.sponsors',
     'wafer.pages',
     'wafer.tickets',
+    # Django isn't finding the overridden templates
+    'registration',
```

Comment:
Having looked more closely at what's happening,, this looks sane, and, as it relies on Django's documentated handling of INSTALLED_APPs, it should be safe enough to land.